api/endpoint-tests/utils.js

```diff
@@ -30,6 +30,27 @@ const login = async (username = 'em@il.com', password = 'password') => {
   throw new Error('Failed to login');
 };
 
+const unauthenticatedTest = (method, url, test) => {
+  test.test('when unauthenticated', async invalidTest => {
+    const { response: { statusCode }, body } = await requestFor(method)(url);
+
+    invalidTest.equal(statusCode, 403, 'gives a 403 status code');
+    invalidTest.notOk(body, 'does not send a body');
+  });
+};
+
+const unauthorizedTest = (method, url, test) => {
+  test.test('with unauthorized', async invalidTest => {
+    const jar = await login('user2@email', 'something'); // this user has no permissions
+    const { response: { statusCode }, body } = await requestFor(method)(url, {
+      jar
+    });
+
+    invalidTest.equal(statusCode, 401, 'gives a 401 status code');
+    invalidTest.notOk(body, 'does not send a body');
+  });
+};
```

Comment:
Since we should have these tests for basically all of our endpoints, I decided to put them in a shared place.  In a future PR, I'll update all the other endpoint tests to include calls to these.  That'll take care of most of the missing endpoint tests.